packages/asset-swapper/src/utils/market_operation_utils/index.ts

```diff
@@ -31,7 +31,7 @@ async function getRfqtIndicativeQuotesAsync(
     marketOperation: MarketOperation,
     assetFillAmount: BigNumber,
     opts: Partial<GetMarketOrdersOpts>,
-): Promise<RfqtIndicativeQuoteResponse[]> {
+): Promise<IndicativeQuote[]> {
```

Comment:
Would it make sense to use `RFQTIndicativeQuote` instead of `IndicativeQuote` type in this file?

Reply:
Addressed in https://github.com/0xProject/0x-monorepo/pull/2582/commits/7162935028a676a93d98e1360be07a7122fb5e56